### ISSUE TITLE
ADD - Juneau, AK

### ISSUE THREAD

**Issue by rsteedle** [defund12/defund12.org#1254]
boroughassembly@juneau.org

Subject: [INSERT UNIQUE SUBJECT LINE]

Message: [MAKE SURE YOU FILL IN BRACKETS. ALSO ADD A LINE OR TWO SO THEY HAVE TO READ EACH MESSAGE]

Dear City and Borough of Juneau Assembly Members,

My name is [YOUR NAME] and I am a resident of [JUNEAU/AUKE BAY/DOUGLAS]. I am writing to demand that CBJ adopts a city budget that prioritizes community well-being, and redirects funding away from the police.

In FY2019, CBJ allocated an egregious $15,130,500 to our police department. Next year, there will likely be deficits as a result of loss of the tourism season. The city may recoup these funds by decreasing the police budget.

I demand that CBJ begin meaningfully defunding the Juneau Police Department and re-allocate those funds to programs proven to more effectively promote a safe and equitable community. We need funding for community-based mental health services, substance abuse treatment services, and affordable housing programs. Police officers should not need to be the first resource for every crisis. I demand a budget that reflects the actual needs of Juneau residents.

History has shown that police “reform” is not enough. Policing in America was developed to protect the institution of slavery. More money and training programs will not only not fix our current system, they will cause future lives to be lost to police brutality. We must take a hard look at the way the current system in place fails to serve-and in fact actively harms-our community, and come together to reimagine the role of police in our city.

I am urging you to keep these things in mind for the budget for the 2020-2021 fiscal year and to invest in the people, not the police.

Thank you for your time,
[YOUR NAME] [YOUR ADDRESS] [YOUR EMAIL] [YOUR PHONE NUMBER]

**Comment by aurendisson** [comment]
Editing notes: I reworded the first paragraph to say "defund" and used the wording from our guide to provide specific examples of community funding. I also updated the police budget figures as reflected in this City and Borough of Juneau document: https://3tb2gc2mxpvu3uwt0l20tbhq-wpengine.netdna-ssl.com/wp-content/uploads/2020/04/FY20-ADOPTED-Budget-Book-FINAL-for-INTERNET.pdf 

Needs work notes: 
I looked for the email addresses of individual council members, but it looks like the city only uses the one listed below for public inquiry, in addition to the online form. I asked in the editing channel about the bracketed all-caps text directly after "Message:" and if this is encouraged. I think we could also use this bracketed area to direct users to the City and Borough of Juneau website to also fill out their form online https://beta.juneau.org/assembly/assembly-contact-form. 

boroughassembly@juneau.org

Subject: [INSERT UNIQUE SUBJECT LINE]

Message: [MAKE SURE YOU FILL IN BRACKETS. ALSO ADD A LINE OR TWO SO THEY HAVE TO READ EACH MESSAGE]

Dear City and Borough of Juneau Assembly Members,

My name is [YOUR NAME] and I am a resident of [JUNEAU/AUKE BAY/DOUGLAS]. I am writing to demand that the City and Borough of Juneau adopt a city budget that defunds the police department and redirects the budget towards social programs and resources that support housing, jobs, education, health care, child care, and other critical community needs.

In FY2019, CBJ allocated an egregious $15,658,000 to our police department and adopted a budget of $16,340,100 for 2020. Next year, there will likely be deficits as a result of our loss of the tourism season. The city may recoup these funds by decreasing the police budget.

I demand that CBJ begin meaningfully defunding the Juneau Police Department and reallocate those funds to programs proven to more effectively promote a safe and equitable community. We need funding for community-based mental health services, substance abuse treatment services, and affordable housing programs. Police officers should not need to be the first resource for every crisis. I demand a budget that reflects the actual needs of the Juneau community.

History has shown that police “reform” is not enough. Policing in America was developed to protect the institution of slavery. More money and training programs will not only not fix our current system, but they will also cause future lives to be lost to police brutality. We must take a hard look at the way the current system in place fails to serve--and in fact, actively harms--our community, and come together to reimagine the role of police in our city.

I am urging you to keep these things in mind for the budget for the 2020-2021 fiscal year and to invest in the people, not the police.

Thank you for your time,
[YOUR NAME] 
[YOUR ADDRESS] 
[YOUR EMAIL] 
[YOUR PHONE NUMBER]
